person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_reader :id, :name, :age
```

Comment:
:name and :age instance variables are the only ones that should be able to be modified that's why they are defined as attr_accessors , kindly fix that 🙏